src/index.js

```diff
@@ -0,0 +1,23 @@
+import './style.css';
+import { addTask } from './modules/addTask.js';
+
+const tasks = [
+  {
+    description: 'wash the dishes',
+    bool: false,
+    index: 1,
+  },
+  {
+    description: 'Complete To Do list project',
+    bool: false,
+    index: 2,
+  },
+];
+
+const addTaskStatic = (tasks) => {
+  tasks.forEach((task) => {
+    addTask(task);
```

Comment:
- [ ] **Required:** Task should be displayed in the order of their index.

![image](https://user-images.githubusercontent.com/27709832/154143501-789a7bcb-d0cc-4487-b49d-cd3a63ee0ac6.png)

**Hint:** Use the sort function. For example `tasks.sort((a, b) => a.index - b.index)`

Reply:
Done